src/test/java/reposense/template/GitTestTemplate.java

```diff
@@ -25,11 +27,16 @@
 import reposense.model.RepoLocation;
 import reposense.parser.InvalidLocationException;
 import reposense.util.FileUtil;
+import reposense.util.TestUtil;
 
 /**
  * Contains templates for git testing.
  */
 public class GitTestTemplate {
+    protected static final Date BLAME_TEST_SINCE_DATE = TestUtil.getSinceDate(2018, Calendar.FEBRUARY, 6);
+    protected static final Date BLAME_TEST_UNTIL_DATE = TestUtil.getUntilDate(2018, Calendar.FEBRUARY, 8);
+    protected static final Date MOVED_FILE_SINCE_DATE = TestUtil.getSinceDate(2018, Calendar.FEBRUARY, 7);
+    protected static final Date MOVED_FILE_UNTIL_DATE = TestUtil.getUntilDate(2018, Calendar.FEBRUARY, 9);
```

Comment:
same for all these, if they are only used in the test class itself, move all to the test class to avoid bloating the template

Reply:
Noted. Have done so.